### PR TITLE
Backend: Expandables

### PR DESCRIPTION
**Pull request by kave**
## Additions

- Backend Expandable moleclue & organism

## Testing

- Create a demo page named `demo` with either Expandable/Expandable groups

## Review

- @anselmbradford @jimmynotjim opened back up, i'd like to merge this PR and focus on the backend page specific JS in 2 individual PR's
- @rosskarchner

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/atoms/hyperlink.html [hunk 1]
```diff
@@ -1 +1,30 @@
...
+
+   hyperlink
+
+   ==========================================================================
+
+   Description:
+
+   Create a hyperlink atom when given:
+
+   value:         An object with the following options for value.
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/atoms/hyperlink.html]: Elsewhere we've been passing in two parameters: `settings` and `options`. The first is for required parameters, the latter is for optional parameters. Would that make sense on the django-side or are conventions for the parameter to always be `value`?

File: cfgov/jinja2/v1/_includes/atoms/hyperlink.html [hunk 2]
```diff
@@ -1 +1,30 @@
...
+        {
+            'hyperlink': {
+                'url' :       '/some-other-url/',
+                'text':  'This is a Link',
+            }
+        })
+    %}
+    {% set value = global_dict.hyperlink %}
+{% endif %}
+<a class='jump-link' href='{{ value.url }}'>{{ value.text }}</a>
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/atoms/hyperlink.html]: Trivial: double-quote HTML attributes.

File: cfgov/jinja2/v1/_includes/molecules/expandable.html [hunk 3]
```diff
@@ -52,10 +63,12 @@
         </div>
     </button>
     <div class="m-expandable_content"
-         {{ 'aria-expanded=true' if settings.is_expanded else '' }}>
+            {{ 'aria-expanded=true' if value.is_expanded else '' }}>
         <div class="m-expandable_content-animated">
-            {{ caller() }}
+            {{ value.paragraph |safe }}
+            {% for item in value.optional_content %}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/expandable.html]: `value.optional_content` is not included in code comment documentation starting on L:14

File: cfgov/jinja2/v1/_includes/molecules/expandable.html [hunk 4]
```diff
@@ -52,10 +63,12 @@
         </div>
     </button>
     <div class="m-expandable_content"
-         {{ 'aria-expanded=true' if settings.is_expanded else '' }}>
+            {{ 'aria-expanded=true' if value.is_expanded else '' }}>
         <div class="m-expandable_content-animated">
-            {{ caller() }}
+            {{ value.paragraph |safe }}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/expandable.html]: Can everything be passed in through `value.optional_content`, including this paragraph? It seems weird to me to have the ability to pass in arbitrary conten, yet separate a paragraph out from that.

File: cfgov/jinja2/v1/_includes/atoms/hyperlink.html (hunk 1 above)
Comment (kave) [inline cfgov/jinja2/v1/_includes/atoms/hyperlink.html]: @anselmbradford conventions for wagtail specifically are all the page values are stored in the template variable `value`

File: cfgov/jinja2/v1/_includes/atoms/hyperlink.html (hunk 2 above)
Comment (kave) [inline cfgov/jinja2/v1/_includes/atoms/hyperlink.html]: donzo

File: cfgov/jinja2/v1/_includes/molecules/expandable.html (hunk 3 above)
Comment (kave) [inline cfgov/jinja2/v1/_includes/molecules/expandable.html]: included

File: cfgov/jinja2/v1/_includes/molecules/expandable.html (hunk 4 above)
Comment (kave) [inline cfgov/jinja2/v1/_includes/molecules/expandable.html]: @anselmbradford yea this is a valid concern. i've merged them into one field

File: cfgov/jinja2/v1/browse-basic/index.html [hunk 5]
```diff
@@ -30,81 +30,74 @@
...
-                    'is_bordered': true,
-                    'is_midtone':  true,
-                    'is_expanded': true
-                }) %}
-                {{ lipsum(1, false, 10, 225) | safe }}
-            {% endcall %}
+            {% include 'molecules/expandable.html' with context %}
+
+
+            {% do global_dict.update(
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/browse-basic/index.html]: Trivial: Follow indenting of https://github.com/cfpb/cfgov-refresh/pull/1251/files#diff-62ffa150f85a8922ad5f88037b0a301bR26

File: cfgov/v1/models/organisms.py [hunk 6]
```diff
@@ -120,3 +120,17 @@ class Meta:
...
+    heading = blocks.CharBlock(required=False)
+    body = blocks.RichTextBlock(required=False)
+
+    is_accordion = blocks.BooleanBlock(required=False)
+    has_rule = blocks.BooleanBlock(required=False)
+
+    expandables = blocks.ListBlock(molecules.Expandable())
+
+    class Meta:
+        icon = 'list-ul'
```
Comment (anselmbradford) [inline cfgov/v1/models/organisms.py]: What does this get converted to? Is the class `list-ul`? That doesn't seem right.

**Comment by anselmbradford** [comment]
Just minor comments :+1:

File: cfgov/v1/models/organisms.py (hunk 6 above)
Comment (kave) [inline cfgov/v1/models/organisms.py]: nah this is just what icon it should use,  when adding the molecule in the streamfields

Comment (anselmbradford) [inline cfgov/v1/models/organisms.py]: Ohhh right... from the limited set.

File: cfgov/jinja2/v1/_includes/atoms/hyperlink.html (hunk 1 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/atoms/hyperlink.html]: :+1:

**Pull request merged**